controllers/host/host_controller.go

```diff
@@ -1730,12 +1730,32 @@ func (r *HostReconciler) ReconcileExistingHost(client *gophercloud.ServiceClient
 
 	inSync := r.CompareAttributes(profile, current, instance, host.Personality)
 
+	if inSync {
+		logHost.Info("Cleaning Delta since inSync")
+		instance.Status.Delta = ""
```

Comment:
Updating the intance may not sufficient, need to consider update the status as line 1744

Reply:
done